### PR TITLE
Document the waxables and oxidizables data maps

### PR DESCRIPTION
**Pull request by Matyrobbrt**
Also add the somewhat recently added `can_villager_compost` field to compostables.

------------------
Preview URL: https://pr-147.neoforged-docs-previews.pages.dev

### CODE REVIEW

File: docs/resources/server/datamaps/builtin.md [hunk 1]
```diff
@@ -98,6 +100,32 @@ Example:
 }
 ```
 
+## `neoforge:oxidizables`
+Allows configuring oxidation stages, as a replacement for `WeatheringCopper#NEXT_BY_BLOCK` (which will be ignored in 1.21.2). This data map is also used to build a reverse deoxidation map (for scraping with an axe). It is located at `neoforge/data_maps/block/oxidizables.json` and its objects have the following structure:
```
Comment (IchHabeHunger54) [inline docs/resources/server/datamaps/builtin.md:104]: New line between heading and paragraph please, and between paragraph and code block as well.

File: docs/resources/server/datamaps/builtin.md [hunk 2]
```diff
@@ -170,5 +198,28 @@ Example:
 }
 ```
 
+## `neoforge:waxables`
```
Comment (IchHabeHunger54) [inline docs/resources/server/datamaps/builtin.md:201]: Again, like above, new line after the heading.

**Review by IchHabeHunger54 — CHANGES_REQUESTED** [review]
Looks good content-wise, two formatting nitpicks.

File: docs/resources/server/datamaps/builtin.md (hunk 1 above)
Comment (Matyrobbrt) [reply docs/resources/server/datamaps/builtin.md:104]:
```suggestion

Allows configuring oxidation stages, as a replacement for `WeatheringCopper#NEXT_BY_BLOCK` (which will be ignored in 1.21.2). This data map is also used to build a reverse deoxidation map (for scraping with an axe). It is located at `neoforge/data_maps/block/oxidizables.json` and its objects have the following structure:
```

File: docs/resources/server/datamaps/builtin.md (hunk 2 above)
Comment (Matyrobbrt) [reply docs/resources/server/datamaps/builtin.md:201]:
```suggestion
## `neoforge:waxables`

```

File: docs/resources/server/datamaps/builtin.md [hunk 3]
```diff
@@ -98,6 +100,33 @@ Example:
 }
 ```
 
+## `neoforge:oxidizables`
+
+Allows configuring oxidation stages, as a replacement for `WeatheringCopper#NEXT_BY_BLOCK` (which will be ignored in 1.21.2). This data map is also used to build a reverse deoxidation map (for scraping with an axe). It is located at `neoforge/data_maps/block/oxidizables.json` and its objects have the following structure:
```
Comment (IchHabeHunger54) [inline docs/resources/server/datamaps/builtin.md:105]:
```suggestion
Allows configuring oxidation stages, as a replacement for `WeatheringCopper#NEXT_BY_BLOCK` (which will be ignored in 1.21.2). This data map is also used to build a reverse deoxidation map (for scraping with an axe). It is located at `neoforge/data_maps/block/oxidizables.json` and its objects have the following structure:

```

**Review by IchHabeHunger54 — CHANGES_REQUESTED** [review]
One final formatting change, then we're good to go.

**Review by IchHabeHunger54 — APPROVED** [review]

**Pull request merged**